default/python/AI/test/test_galaxy_setup_data_wrapper.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from copy import deepcopy
+
+import freeOrionAIInterface as fo
+
+from abstract_test_bases import PropertyTester
+from utils import greater_or_equal, is_not_equal
+
+TYPE = PropertyTester.TYPE
+READ_ONLY = PropertyTester.READ_ONLY
+ADDITIONAL_TESTS = PropertyTester.ADDITIONAL_TESTS
+
+
+class GalaxySetupDataTester(PropertyTester):
+    class_to_test = fo.GalaxySetupData
+    properties = deepcopy(PropertyTester.properties)
+    properties.update({
+        "seed": {
+            TYPE: str,
+        },
+        "size": {
+            TYPE: int,
+            ADDITIONAL_TESTS: [greater_or_equal(1)],
+        },
+        "shape": {
+            TYPE: fo.galaxyShape,
+            ADDITIONAL_TESTS: [is_not_equal(fo.galaxyShape.invalid)],
+        },
+        "age": {
+            TYPE: fo.galaxySetupOption,
+            ADDITIONAL_TESTS: [is_not_equal(fo.galaxyShape.invalid)],
```

Comment:
You use constant from one class to test result of the other.  `fo.galaxyShape.invalid`  for `fo.galaxySetupOption`.  Same in other tests.